decentralised-identity.md

```diff
@@ -0,0 +1,157 @@
+
+Imagine if your ID card only worked with one type of door lock. That would be inconvenient, right? Interoperability solves this issue. It means your decentralized identity can be recognized and used on different apps, websites, or services, just like your ID works wherever you go.
+
+Imagine buying things online without constantly entering your details. Using blockchain, your payment details become tamper-resistant and transparent. No single authority holds all your financial info, reducing the risk of a massive data breach. This isn't just about security; it's about convenience. With a decentralized identity, online payments become more streamlined, less prone to fraud, and entirely under your control. 
+
+The key is having common standards and rules that everyone follows. This way, whether you're logging into a social media account, accessing a financial service, or using an online marketplace, your decentralized identity seamlessly fits. It's like having a universal key for your digital interactions, making things more convenient and user-friendly. Interoperability is the glue that holds together the vision of a decentralized identity.
+
+### Would security be an issue?
+
+Security remains a critical consideration, primarily as these rely heavily on technologies like blockchain and cryptographic methods. Potential vulnerabilities may arise from implementation flaws, coding errors, or attacks on the underlying infrastructure. The use of cryptographic keys, while fundamental for user control, introduces risks if not adequately managed, with potential concerns related to key loss or theft. Additionally, the distributed and decentralized nature of these systems may pose challenges in ensuring uniform security standards across the entire network.
+
+### Understanding Single Points of Failure and the potential for Eliminating or Not: 
+
+Getting rid of single points of failure in decentralized identity is like making sure there's no one weak link that can mess things up. Think of it as spreading your eggs across different baskets instead of putting them all in one.
+
+In traditional systems, there's often one central authority holding all the keys to your identity. If that authority gets compromised, it's like opening Pandora's box – a single breach can lead to a massive data disaster. Decentralized identity avoids this risk by spreading the responsibility. There's no single database or authority holding all the crucial info. Instead, it's distributed across a network of computers, making it much harder for a single mishap to cause a widespread identity catastrophe.
+
+This approach aligns with the cypherpunk belief in reducing dependency on centralized powers. Eliminating single points of failure means your digital identity isn't hanging by a thread; it's distributed, making the whole system more resilient and secure. It's like having a safety net for your online identity, ensuring that even if one part falters, the rest stand strong.
```

Comment:
If the above is true (line 109). This also brings into question this statement. Would a cypherpunk really want their data stored on a blockchain so that one day it could be exposed to the world? I think the answer is no. Cypherpunks would advocate for a system where no personal data was leaked. 

Basic OPSEC suggests the data you share should be separate out into as main fragments as possible e.g.

* Never share personal data on the internet
* Financial data - should be kept in separate accounts and wallets so no-one can track your wealth
* Internet data - a different clocked-email, user names, and password, should be used for every website
* and so on ...

This is yet another reason I'm an advocate for an account-less internet. e.g. I have a wallet and key pair that is just for internet connectivity and to access a website the only data I'm revealing is a public key. Which probably should be different for each service.

Reply:
Yes this is a valid point. I've taken out the paragraph 'This approach aligns with the cypherpunk belief....'